src/views/Dashboard.vue

```diff
@@ -392,4 +394,9 @@ section header {
 .pending-tooltip {
   max-width: 16.5rem;
 }
+
+.section-count {
+  color: $gray9;
+  font-weight: 500;
```

Comment:
Our fonts only support normal and bold (400 and 700), so stick to those weights please.

Reply:
Or use `font-weight: bold` here.

Or use `font-weight-bold` class in element itself.

Reply:
The ticket asks for the numbers to be regular font weight, so I'll change it to font-weight: normal

Reply:
It's bolded otherwise?

OK.

Reply:
Yeah, so I guess it's supposed to be (for example) **To Do** (1), where To Do is bold and the number (1) is regular.